### PR TITLE
[T4A4][T13-B1] Nathan Ohun Kwon 

### PR DESCRIPTION
**Pull request by kwonn**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface to print contact details.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:4]: :+1:  for header comments.
can you think of making it more informative?

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+ */
+
+public interface Printable {
+
+    String getPrintableString();
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:9]: think of a suitable class where this method can to be implemented to return the printable string for all members passed.

**Comment by okkhoy** [comment]
@kwonn 
Some comments added. Please ack & close the PR after reading comments.

**Comment by kwonn** [comment]
Thank you!

**Pull request closed without merging**